### DISCUSSION TITLE
50 mb File upload

### DISCUSSION THREAD

**Discussion by nithinkadam19** [artilleryio/artillery#3322] [Artillery]
Hi Team,

I have a scenario where i need to run load test for 60mb file. Is it possible to upload and download 60MB files using artillery ?
If so what is the procedure.

**Comment by hassy** [comment]
It should just work. Have you run into any issues?

Docs on file uploads here: https://www.artillery.io/docs/reference/engines/http#uploading-files

**Comment by nithinkadam19** [comment]
When i run with 50mb file i am getting entity is too large.
we converted pdf to base-64

**Reply by hassy** [reply]
> The HTTP 413 Content Too Large [client error response](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses) status code indicates that the request entity was larger than limits defined by server.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413

Sounds like the service you're testing has an upload limit lower than 50 MB.

**Comment by nithinkadam19** [comment]
No, We have not specified any content length. we have only converted 50mb pdf file to base64.

**Reply by hassy** [reply]
Do you have a reason to think that the error comes from Artillery, rather than from the service you're testing? Have you tried uploading the exact same file to the application using something else? (e.g. directly from the browser) Can't help you unless you provide some more information. No mind readers around here.
